calculator.js

```diff
@@ -1,133 +1,133 @@
+				maximumFractionDigits: 0,
+			});
+		}
+		if (decimalDigits != null) {
+			return `${integerDisplay}.${decimalDigits}
     `;
-    } else {
-      return integerDisplay;
-    }
-  }
-  updateDisplay() {
-    this.currentOperandTextElement.innerText = this.getDisplayNumber(
-      this.currentOperand
-    );
-    if (this.operation != null) {
-      this.previousOperandTextElement.innerText = `${this.getDisplayNumber(
-        this.previousOperand
-      )}${this.operation}`;
-    } else {
-      this.previousOperandTextElement.innerText = "";
-    }
-  }
+		} else {
+			return integerDisplay;
+		}
+	}
+	updateDisplay() {
+		this.currentOperandTextElement.innerText = this.getDisplayNumber(
+			this.currentOperand
+		);
+		if (this.operation != null) {
+			this.previousOperandTextElement.innerText = `${this.getDisplayNumber(
+				this.previousOperand
+			)}${this.operation}`;
+		} else {
+			this.previousOperandTextElement.innerText = "";
+		}
+	}
 }
 const numberButtons = document.querySelectorAll("[data-number]");
 const operationButtons = document.querySelectorAll("[data-operation]");
 const equalsButton = document.querySelector("[data-equals]");
 const deleteButton = document.querySelector("[data-delete]");
 const allClearButton = document.querySelector("[data-all-clear]");
 const previousOperandTextElement = document.querySelector(
-  "[data-previous-operand]"
+	"[data-previous-operand]"
 );
 const currentOperandTextElement = document.querySelector(
-  "[data-current-operand"
+	"[data-current-operand"
 );
 
 const calculator = new Calculator(
-  previousOperandTextElement,
-  currentOperandTextElement
+	previousOperandTextElement,
+	currentOperandTextElement
 );
 
 numberButtons.forEach((button) => {
-  button.addEventListener("click", () => {
-    calculator.appendNumber(button.innerText);
-    calculator.updateDisplay();
-  });
+	button.addEventListener("click", () => {
+		calculator.appendNumber(button.innerText);
+		calculator.updateDisplay();
+	});
 });
 operationButtons.forEach((button) => {
-  button.addEventListener("click", () => {
-    calculator.appendNumber(button.innerText);
-    calculator.updateDisplay();
-  });
+	button.addEventListener("click", () => {
+		calculator.appendNumber(button.innerText);
+		calculator.updateDisplay();
+	});
 });
 
 operationButtons.forEach((button) => {
-  button.addEventListener("click", () => {
-    calculator.chooseOperation(button.innerText);
-    calculator.updateDisplay();
-  });
+	button.addEventListener("click", () => {
+		calculator.chooseOperation(button.innerText);
+		calculator.updateDisplay();
+	});
 });
 equalsButton.addEventListener("click", (button) => {
-  calculator.compute();
-  calculator.updateDisplay();
+	calculator.compute();
+	calculator.updateDisplay();
 });
 allClearButton.addEventListener("click", (button) => {
-  calculator.clear();
-  calculator.updateDisplay;
+	calculator.clear();
+	calculator.updateDisplay();
```

Comment:
Without parenthesis this was just a reference to a function rather than a function call.